bot/resources/tags/return.md

```diff
@@ -1,35 +1,32 @@
 **Return Statement**
 
-When calling a function, you'll often want it to give you a value back. In order to do that, you must `return` it. The reason for this is because functions have their own scope. Any values defined within the function body are inaccessible outside of that function.
-
-*For more information about scope, see `!tags scope`*
+A value created inside a function can't be used outside of it unless you `return` it.
 
 Consider the following function:
 ```py
 def square(n):
-    return n*n
+    return n * n
 ```
-If we wanted to store 5 squared in a variable called `x`, we could do that like so:
+If we wanted to store 5 squared in a variable called `x`, we would do:
 `x = square(5)`. `x` would now equal `25`.
 
 **Common Mistakes**
 ```py
 >>> def square(n):
-...     n*n  # calculates then throws away, returns None
+...     n * n  # calculates then throws away, returns None
 ...
 >>> x = square(5)
 >>> print(x)
 None
 >>> def square(n):
-...     print(n*n)  # calculates and prints, then throws away and returns None
+...     print(n * n)  # calculates and prints, then throws away and returns None
 ...
 >>> x = square(5)
 25
 >>> print(x)
 None
 ```
 **Things to note**  
-• `print()` and `return` do **not** accomplish the same thing. `print()` will only print the value, it will not be accessible outside of the function afterwards.  
-• A function will return `None` if it ends without reaching an explicit `return` statement.  
-• When you want to print a value calculated in a function, instead of printing inside the function, it is often better to return the value and print the *function call* instead.  
-• [Official documentation for `return`](https://docs.python.org/3/reference/simple_stmts.html#the-return-statement)  
+• `print()` and `return` do **not** accomplish the same thing. `print()` will show the value, and then it will be gone.  
+• A function will return `None` if it ends without a `return` statement.  
+• When you want to print a value from a function, it's best to return the value and print the *function call* instead, like `print(square(5))`.  
```

Comment:
```suggestion
• When you want to print a value from a function, it's best to return the value and print the *function call* instead, for example, `print(square(5))`.  
```

Reply:
I am indifferent to this change, but I would prefer "instead; for example," instead of "instead, for example".

Reply:
I think "like" is fine, and prefer anything that is shorter